docker/importer/volume/surveillance-mediator/endpoint-resource-resolver-dhis.json

```
@@ -0,0 +1,100 @@
+{
+  "name": "Covid19 DHIS2 Orchestrator",
+  "description": "Endpoint for mapping the Covid19 case report into DHIS format and then forwarding it to DHIS2",
+  "endpoint": {
+    "pattern": "/resource-resolver-dhis"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "inputValidation": {
+    "type": "object",
+    "properties": {
+      "requestBody": {
+        "type": "object"
+      }
+    },
+    "required": ["requestBody"]
+  },
+  "inputMapping": {
+    "requestBody.identifier": "teiIdentifier",
+    "constants.program": "program",
+    "constants.orgUnit": "orgUnit",
+    "constants.trackedEntityType": "trackedEntityType",
+    "constants.status": "event.status",
+    "constants.stage": "event.programStage",
+
+    "transforms.patinfo_ID": "attributes[]+",
+    "transforms.patinfo_ageonset": "attributes[]+",
+    "transforms.patinfo_sex": "attributes[]+",
+    "transforms.caseId": "attributes[]+",
+
+    "transforms.incidentDate": {
+      "key": "incidentDate",
+      "transform": {
+        "function": "dateTimeToDate"
+      }
+    },
+
+    "transforms.patcourse_asymp": "event.dataValues[]+",
+    "transforms.Comcond_any": "event.dataValues[]+",
+    "transforms.patcourse_dateonset": "event.dataValues[]+",
+    "transforms.expo_travel": "event.dataValues[]+"
+  },
+  "inputTransforms": {
+    "patinfo_ID": "{ 'attribute': constants.attributes.patinfo_ID, 'value': requestBody.identifier }",
```

Incorrect nesting
```suggestion
    "patinfo_ID": "{ 'attribute': constants.attributes.patinfo_ID, 'value': requestBody.data.identifier }",
```

Same for the other fields using requestBody

---

addressed